package/gluon-status-page/src/js/lib/gui/nodeinfo.js

```diff
@@ -39,8 +50,8 @@ define(["lib/helper"], function (Helper) {
       dlEntry(list, nodeInfo, "network.mac", Helper._("Primary MAC"))
       dlEntry(list, nodeInfo, "network.addresses", Helper._("IP Address"))
       dlEntry(list, nodeInfo, "software.firmware.release", Helper._("Firmware"))
-      dlEntry(list, nodeInfo, "software.fastd.enabled", "Mesh-VPN")
-      dlEntry(list, nodeInfo, "software.autoupdater.enabled", Helper._("Automatic updates"))
+      dlEntry(list, nodeInfo, "software.fastd.enabled", "Mesh-VPN", enabledDisabled)
```

Comment:
If we're touching this, Mesh-VPN should get localized as well. The English version should be "Mesh VPN".